src/app/resources/iis_scripts/admin_api_util.ps1

```diff
@@ -245,34 +246,17 @@ if (!$config.security.users.owners.Contains($user)) {
     }
 }
 
-$administratorsSID = 'S-1-5-32-544';
 if ($saveConfig) {
     LogVerbose "Saving config..."
     if ($devMode) {
         throw "Cannot edit config file in dev mode, please add $user to ""$iisAdminOwners"" group manually"
     }
-    ## Check the current user is a member of the Administrators local user group
-    ## NOTE: Get-LocalGroupMember works case-sensitevly. So, as a workaround, Where-Object is used here
-    if (!(Get-LocalGroupMember -SID $administratorsSID -ErrorAction SilentlyContinue | Where-Object { $_.Name -eq $user } )) {
-        throw "User $user lacks administrator privilege which is needed to initiate Microsoft IIS Administration API"
-    }
     $apiHome = [System.IO.Path]::Combine($workingDirectory, "..")
     ## Installer added a read-only rule on current user to the directory, delete it
     $dirAcl = Get-Acl $apiHome
-    foreach ($access in $dirAcl.Access | Where-Object { $_.IdentityReference.Value -eq $user }) {
-        $dirAcl.RemoveAccessRule($access) | Out-Null
-        $updateAcl = $true
-    }
-    if ($updateAcl) {
-        LogVerbose "Updating Acl to remove readonly access..."
-        Set-Acl -Path $apiHome -AclObject $dirAcl | Out-Null
-    }
-
     $modifyAcess = [System.Security.AccessControl.FileSystemRights]::Modify
     $allow = [System.Security.AccessControl.AccessControlType]::Allow
-    $builtInAdministrators = (ConvertTo-NTAccount $administratorsSID).value

```

Comment:
The function ConvertTo-NTAccount() is not used anymore. Please remove the function because we don't need that.